test/overlap.jl

```diff
@@ -63,3 +64,9 @@ end
 
 
 end
+
```

Comment:
I can see that some tests for the remaining functions are missing, it would be great if you can them in this PR!